tests/test_validators.py

```diff
@@ -987,15 +1052,14 @@ def root_validator(self, values):
                 return values
 
 
-@pytest.mark.xfail(reason='working on V2')
 def test_root_validator_extra():
-    with pytest.raises(errors.PydanticUserError) as exc_info:
+    with pytest.raises(TypeError, match=r'Unsupported signature for after validator') as exc_info:
```

Comment:
Not sure if the train has already left the station, but I would really prefer it somehow make it clear that it means a `mode='after'`-validator; right now trying to put myself in the shoes of a new-to-v2 user, I read `Unsupported signature for after validator` and think it's a signature coming after a validator or something else I don't understand.

Reply:
Since we now have `@validator` and `@field_validator` using `@validator` always warns and just tells you to use V2 validators.